sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/spanner/changestreams/dao/ChangeStreamDao.java

```diff
@@ -127,6 +129,7 @@ public ChangeStreamResultSet changeStreamQuery(
               .to(heartbeatMillis)
               .build();
     }
+    LOG.info("Change Stream Statement: [{}] ", statement.toString());
```

Comment:
in general refrain logging PII (statements can contain PII). If needed could use debug log